setup.py

```diff
@@ -14,6 +14,7 @@
 
 install_requires = [
     'draftjs_exporter==0.7.0',
+    'jinja2>=2.9.5,<3'  # TODO: remove when it's not necessary (https://github.com/springload/wagtaildraftail/issues/13)
```

Comment:
Oh, apparently the equivalent [Django Template Tag](https://github.com/springload/wagtaildraftail/blob/758254139cebb6e6c3d3d8db731615772a2f535e/wagtaildraftail/templatetags/draftail_tags.py) has already been created so `DraftailTextBlock` should use this instead. Then if you make sure that `wagtaildraftail.filters` is never imported (and later converted to something similar to [this](https://github.com/wagtail/wagtail/blob/1874b8b259e1295370b1c2122966d96668db7eec/wagtail/wagtailcore/jinja2tags.py#L10)) and the dependency can be removed.